### PR TITLE
[don't merge] Vara: Rolling changes

### PR DESCRIPTION
**Pull request by NikVolf**
This is to keep CI running on vara branch

### CODE REVIEW

File: scripts/gear.sh [hunk 1]
```diff
@@ -315,6 +315,10 @@ case "$COMMAND" in
         header "Running gclient tests"
         client_tests "$ROOT_DIR" "$@"; ;;
 
+      uploads)
```
Comment (breathx) [inline scripts/gear.sh:318]: Dead test

File: scripts/test-utils/harmlessUploadDemos.js [hunk 2]
```diff
@@ -0,0 +1,60 @@
+const { ApiPromise, WsProvider, Keyring } = require('@polkadot/api');
```
Comment (breathx) [inline scripts/test-utils/harmlessUploadDemos.js:1]: Dead test

File: scripts/test-utils/infiniteLoop.js [hunk 3]
```diff
@@ -0,0 +1,95 @@
+const { ApiPromise, WsProvider, Keyring } = require('@polkadot/api');
+const { randomAsHex } = require('@polkadot/util-crypto');
```
Comment (breathx) [inline scripts/test-utils/infiniteLoop.js:2]: Dead test

File: runtime/gear/src/lib.rs [hunk 4]
```diff
@@ -97,7 +97,7 @@ pub const VERSION: RuntimeVersion = RuntimeVersion {
     impl_name: create_runtime_str!("gear"),
     apis: RUNTIME_API_VERSIONS,
     authoring_version: 1,
-    spec_version: 100,
+    spec_version: 130,
```
Comment (breathx) [inline runtime/gear/src/lib.rs:100]: I'm not sure about setting specific versions for vara commit itself

**Review by breathx — CHANGES_REQUESTED** [review]
This files are also deprecated: 
- pallets/gear-program/weights/pallet_gear_program.rs
- program/api/generated/metadata.rs

**Comment by NikVolf** [comment]
@breathx feel free to update those, this is not an actual PR :)

**Review by ukint-vs — APPROVED** [review]

**Review by breathx — APPROVED** [review]

**Pull request merged**